.github/workflows/publish.yml

```diff
@@ -18,8 +18,12 @@ jobs:
           visitorId: '${{ secrets.VISITOR_ID }}'
           privateKey: '${{ secrets.PRIVATE_KEY }}'
           requestId: '${{ secrets.REQUEST_ID }}'
-      - name: Deploy 🚀
+      - name: Deploy to GitHub pages 🚀📘
         uses: JamesIves/github-pages-deploy-action@8817a56e5bfec6e2b08345c81f4d422db53a2cdc
         with:
           branch: gh-pages
           folder: dist
+      - name: Deploy to Readme API Reference 🚀🦉
+        uses: readmeio/rdme@v8
```

Comment:
Use commit SHA instead of tag for security reasons please

Reply:
Done, thanks :)